coap/tracing/tracing.go

```diff
@@ -0,0 +1,51 @@
+package tracing
```

Comment:
This shouldn't be there since we are doing tracing middleware on nats when we publish and subscribe